app/controllers/v1/feature_toggles_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module V1
+  class FeatureTogglesController < ApplicationController
+    skip_before_action :authenticate
+    before_action :load_user
+
+    def index
```

Comment:
If we allow functionality to query just a single feature (which IMO is the best thing we could do to make this more efficient in general), I think we should put that functionality in a `show` route in this controller, to split it from the full `index` functionality